x-pack/plugins/stack_connectors/server/connector_types/cases/jira/service.ts

```diff
@@ -136,6 +136,9 @@ export const createExternalService = (
     }, '');
   };
 
+  const escapeSpecialCharacters = (str: string) =>
```

Comment:
I know it's generally not an ideal solution to expose internal functions just for testing but could we do that here? Regex always makes me scared haha 😨 so it'd be good to have some tests around this function directly. It'll be way easier to test if we just export it from this file tool.

Reply:
Could we also add a comment that includes this link: https://confluence.atlassian.com/jirasoftwareserver/search-syntax-for-text-fields-939938747.html

maybe something like:

> These characters need to be escaped per Jira's search syntax, see <url> for more details.

Reply:
added comment in [a73197](https://github.com/elastic/kibana/pull/145610/commits/a73197ef9e85657180c33cd6cee4290f9b8e655a)

Reply:
> I know it's generally not an ideal solution to expose internal functions just for testing but could we do that here? Regex always makes me scared haha 😨 so it'd be good to have some tests around this function directly. It'll be way easier to test if we just export it from this file tool.

Shall I put this function in a `x-pack/plugins/stack_connectors/server/connector_types/cases/jira/utils.ts` for better access and tests?

Reply:
Yeah sure!

Reply:
fixed in [f060450](https://github.com/elastic/kibana/pull/145610/commits/f060450a8efe3d8ad53d462be46d11aa82db8158)